RELEASE.md

```diff
@@ -1,8 +1,15 @@
 ## Release Finalization
 
 To cut a release:
+1. Ensure the relevant spec APIs have been released
+1. Ensure any [migration guides](https://github.com/buildpacks/docs/tree/main/content/docs/reference/spec/migration) have been created.
+1. If applicable, ensure the README is updated with the latest supported apis (example PR: https://github.com/buildpacks/lifecycle/pull/550)
 1. Create a release branch in the format `release/0.99.0`. New commits to this branch will trigger the `build` workflow and produce a lifecycle image: `buildpacksio/lifecycle:<commit sha>`.
-1. If applicable, ensure the README is updated with the latest supported apis (example PR: https://github.com/buildpacks/lifecycle/pull/509/files)
 1. When ready to cut the release, manually trigger the `draft-release` workflow: Actions -> draft-release -> Run workflow -> Use workflow from branch: `release/0.99.0`. This will create a draft release on GitHub using the artifacts from the `build` workflow run for the latest commit on the release branch.
-1. Edit the release notes as necessary and perform any manual validation of the artifacts
+1. Edit the release notes as necessary
+1. Perform any manual validation of the artifacts
```

Comment:
```suggestion
1. Perform any manual validation of the artifacts
Inside the `pack` repo:
```

Reply:
(otherwise it might seem that we're running `make acceptance` from the lifecycle repo)